models/reset_password/reset_password.go

```diff
@@ -16,27 +16,27 @@ type ResetPassword interface {
 }
 
 type ResetPasswordStruct struct {
-	Id        int64
-	UserId    int64
+	ID        int64
+	UserID    int64
 	Token     string
 	ExpiresAt time.Time
 	database  db.DB
 }
 
-func NewResetPassword(id int64, userId int64, token string, expiresAt time.Time) *ResetPasswordStruct {
-	resetPassword := &ResetPasswordStruct{Id: id, UserId: userId, Token: token, ExpiresAt: expiresAt}
+func NewResetPassword(id int64, userID int64, token string, expiresAt time.Time) *ResetPasswordStruct {
+	resetPassword := &ResetPasswordStruct{ID: id, UserID: userID, Token: token, ExpiresAt: expiresAt}
 	resetPassword.Initialize()
 	return resetPassword
 }
 
-func GenerateResetPassword(userId int64, email string) *ResetPasswordStruct {
+func GenerateResetPassword(userID int64, email string) *ResetPasswordStruct {
```

Comment:
__[golint]__
* [golint] exported function GenerateResetPassword should have comment or be unexported